### DISCUSSION TITLE
jellyseerr (docker) not able to save anything anymore after docker restart

### DISCUSSION THREAD

**Discussion by Wurzelmann** [seerr-team/seerr#1024] [Q&A] [answered]
Yesterday I tried the docker version of jellyseerr for the first time (because I had trouble building the latest version from source) and it worked perfectly up until the server or docker gets restarted. Afterwards, it is not able to save config changes, request changes or anything else.

I cannot find the source of the problem and I find it rather curious. It's not a permission problem because it can save all the things in its defined folder's config. What am I missing here? :-/

Docker config is taken from the official documentation: https://docs.jellyseerr.dev/getting-started/docker, running behind nginx as reverse proxy.

**Comment by fallenbagel** [comment]
Please hop onto discord for further troubleshooting. This sounds like an issue with csrf which we recently fixed but I need some logs and to see what the actual issue first and its easier to debug on discord

**Comment by fallenbagel** [comment, marked as answer]
Oh wait I just read what you wrote. "Yesterday"
Yeah the issue is the csrfProtection when you have that enabled it was not working with https. This has been fixed in `2.0.1` ensure you're on `2.0.1` by doing `docker pull fallenbagel/jellyseerr:latest` or use the tag `v2.0.1`

For reference here's the issue:
#1011

**Comment by Wurzelmann** [comment]
Oh, you're correct, that was it! I really picked the wrong day to move from building it myself to the Docker version. :grimacing:  Thank you very much for the quick reply, works perfectly now, I appreciate it!
